docs/examples/tutorial/pure/c_arrays.pyx

```diff
@@ -0,0 +1,12 @@
+def count_digits(digits):
+    cdef int counts[10]  

```

Comment:
We prefer java-based array declaration - ref. https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#c-arrays